modules/swagger-codegen/src/main/resources/cpprest/model-source.mustache

```diff
@@ -2,6 +2,7 @@
 {{#models}}{{#model}}
 
 #include "{{classname}}.h"
+#include "Object.h"
```

Comment:
why reinclude it? you already included it in the header of this

Reply:
Also, does every single model really need an Object?

Reply:
You're right, I missed that, I removed the unnecessary include. Also I haven't found a way to only include the Object header under certain circumstances.